percy/connection.py

```diff
@@ -7,6 +9,28 @@ def __init__(self, config, environment):
         self.config = config
         self.user_agent = str(UserAgent(config, environment))
 
+    def _requests_retry_session(
+        self,
+        retries=3,
+        backoff_factor=0.3,
+        method_whitelist=['HEAD', 'GET', 'POST', 'PUT', 'DELETE', 'OPTIONS', 'TRACE'],
+        status_forcelist=(500, 502, 503, 504, 520),
```

Comment:
for what it's worth, one of our requests to percy failed with a `524 Server Error: Origin Time-out`

Reply:
Ha, thanks @teeberg!  I knew there was another 52X code I needed to add, but hadn't looked up the precise one yet.  Will add 524 now.

Reply:
that was fast :-) thanks!